src/Elastic.Apm/Api/Request.cs

```diff
@@ -0,0 +1,22 @@
+namespace Elastic.Apm.Api
+{
+	/// <summary>
+	/// Encapsulates Request related information that can be attached to an <see cref="ITransaction" />.
+	/// See <see cref="ITransaction.Request" />
+	/// </summary>
+	public class Request

```

Comment:
Hi @gregkalapos , maybe you can add Body field to the object please ?

Reply:
Good catch, thanks! 

Fixed.

Reply:
Thanks @gregkalapos